app/models/bundle_context.rb

```diff
@@ -39,6 +47,14 @@ def progress_log_file
     Tempfile.new.path(id) #FIXME: (#78)
   end
 
+  def stageable_discovery
+    {}
+  end
```

Comment:
effectively empty method worries me - perhaps addressed in subsequent commit?

Reply:
#274